inst/examples/pmx_nm.R

```diff
@@ -0,0 +1,26 @@
+\donttest{
+
+## using only runnumber 
+ctr <- pmx_nm(
+  directory=model_dir,
+  runno = "001" 
+)
+
+## using a model file (e.g. run001.lst)
+ctr <- pmx_nm(
+  directory=model_dir,
+  file = "run001.lst" 
+)
+
+## if simulation was performed post-hoc, an additional simulation file can be loaded for VPC
+ctr <- pmx_nm(
+  directory=model_dir,
+  file = "run001.lst", 
+  simfile = "simulation.ctl"
+)
+
+## loading with individual table(s)-names
+ctr <- pmx_nm(directory = model_dir,
+              runno = 3, 
+              table_names = "xptab")
+}
```

Comment:
Why is this file here?  Usually examples are in vignettes or in documentation.  This is entirely outside and still contains some `Rd` markdown.  Is this a mistake?

Reply:
Other examples of documentation of other functions are also in this folder. I didn't want to change to much in the structure, we might discuss this